src/auth/auth.ts

```diff
@@ -36,6 +39,26 @@ export class BBAuth {
     }
   } = {};
 
+  public static getUri(
```

Comment:
I know we talked about changing the parameter from `url` to `uri`, but I think this method and the `args` type should still say `url` since the input is a 1bb-specific URI and the output is a web address/URL.